moveit_ros/planning_interface/move_group_interface/src/wrap_python_move_group.cpp

```diff
@@ -536,14 +536,16 @@ class MoveGroupInterfaceWrapper : protected py_bindings_tools::ROScppInitializer
     }
   }
 
-  Eigen::MatrixXd getJacobianMatrixPython(bp::list& joint_values)
+  Eigen::MatrixXd getJacobianMatrixPython(bp::list& joint_values, bp::list& reference_point_position)
```

Comment:
To correctly handle the issue in https://github.com/ros-planning/moveit/pull/1595#issuecomment-519434052, reference_point_position needs to be optional here as well.
That's because this method can be called directly as well.

Reply:
I don't know how to initialize a list to `[0.0, 0.0, 0.0]` for a default argument, but the new commit should work too. Thanks for taking another look.

Reply:
I implemented this.